### DISCUSSION TITLE
Partial dispersion parameter

### DISCUSSION THREAD

**Discussion by dibyendumajumdar** [mjhoptics/ray-optics#35] [Q&A] [answered]
Re the difficulty I had with the ray tracing of glasses with partial dispersion, the patent literature often has a partial dispersion parameter in addition to the abbe number and refractive index. Can this be used by ray-optics? Example of a patent with two elements that have partial dispersion:

![apo-lanthar-50mm-f2-specs](https://user-images.githubusercontent.com/1548494/111903002-f731c300-8a37-11eb-9369-89c558dc4fe0.JPG)

**Comment by mjhoptics** [comment]
This could potentially be helpful. The example above specifies the partial between F (486nm) and g (436nm), as well as Nd and Vd. If the partial were from d (587nm) to F, we'd have enough data to get 3 wavelength/index pairs (d, C, F), but with the F/g partial I can only get the index differences between C, F and g. This *might* be enough to do a 3 term Hartmann fit, but I haven't tried that.

So conceptually, yes, we could create a new glass class that took nd, Vd and PFd (or possibility PgF) as input, fit a Hartmann polynomial to it and hopefully see a better model for ED glasses.

**Comment by dibyendumajumdar** [comment, marked as answer]
I guess probably not worth it ... As if we have exact glass types that is better anyway.
